Catalog/src/Application/Category/Commands/DeleteCategory/DeleteCategoryCommand.cs

```diff
@@ -20,7 +20,7 @@ public DeleteCategoryCommandHandler(IApplicationDbContext context)
 
     public async Task<int> Handle(DeleteCategoryCommand request, CancellationToken cancellationToken)
     {
-        var entity = _context.Categories.FirstOrDefault(c => c.Id == request.CategoryId);
+        var entity = _context.Categories.FirstOrDefault(c => c.Id == request.Id);

```

Comment:
Can you confirm whether items are also deleted?
from requirements: Delete category (with the related items)

Reply:
Confirm. 
Foreign key was created with DELETE CASCADE option.
CONSTRAINT [FK_Products_Categories_CategoryId] FOREIGN KEY ([CategoryId]) REFERENCES [dbo].[Categories] ([Id]) ON DELETE CASCADE